app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/SpecifyShopType.kt

```diff
@@ -0,0 +1,64 @@
+package de.westnordost.streetcomplete.quests.shop_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.LAST_CHECK_DATE_KEYS
+import de.westnordost.streetcomplete.data.meta.SURVEY_MARK_KEY
+import de.westnordost.streetcomplete.data.meta.toCheckDateString
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.ktx.containsAny
+import java.util.*
+
+class SpecifyShopType : OsmFilterQuestType<ShopTypeAnswer>() {
+
+    override val elementFilter = """
+        nodes, ways, relations with (
+             shop = yes
+             and !man_made
+             and !historic
+             and !military
+             and !power
+             and !tourism
+             and !attraction
+             and !amenity
+             and !leisure
```

Comment:
Are these needed? The quest does not enable POI deletion, so this shouldn't be an issue.

Reply:
Some people (IMHO badly and pointlessly) used `amenity=fuel` `shop=yes` to tag "this fuel station has shop".

In theory the same mistake can be applied more widely, so this avoids all of that.

(I prefer overkill over whack-a-mole in case of handling tagging schemes, but if someone wants to exclude only `amenity=fuel` I would not protest)